sites/rcvfortexas-1/donate_3.html

```diff
@@ -0,0 +1,391 @@
+<div id="content">
+
+  {% if page.headline.size > 0 %}
+  	<div id="headline">
+  	  <h2>{{ page.headline }}</h2>
+  	</div>
+  {% endif %}
+  <p class="donate-type-intro">Use my donation for:</p>
+  <div class="donate-type-selector">
+    <div>
+      <a class="donate-link-noshadow" href="/donate">
+      <div class="donate-type-button">
+        <div class="donate-type-heading">Education &amp; Outreach</div>
+        <div>Tax-deductible</div>
+        <div>501(c)(3)</div>
+      </div>
+      </a>
+    </div>
+    <div>
+      <div class="donate-type-button active">
+        <div class="donate-type-heading">Legislative Advocacy</div>
+        <div>Not tax-deductible</div>
+        <div>501(c)(4)</div>
+      </div>
+    </div>
+  </div>
```

Comment:
It looks like this is the only difference between this file and `donate.html`.  The fault's obviously with the way NationBuilder sets things up, but the code duplication still bugs me. Would it be possible to shift everything else into a file called something like `_donate.html` and then import it here with `{% include "donate" %}`?